ext/drreg/drreg.c

```diff
@@ -867,7 +877,7 @@ drreg_reserve_register(void *drcontext, instrlist_t *ilist, instr_t *where,
     dr_pred_type_t pred = instrlist_get_auto_predicate(ilist);
     drreg_status_t res;
     if (drmgr_current_bb_phase(drcontext) != DRMGR_PHASE_INSERTION) {
-        res = drreg_forward_analysis(drcontext, where);
+        res = drreg_forward_liveness_analysis(drcontext, where);
```

Comment:
nit: Not sure about this renaming: that function does other things than just liveness doesn't it?  It counts app uses, thought there was sthg else -- well at least one other thing, which then makes the new name seem worse than the old to me.

Reply:
Though if both of you prefer a name change then ok.

Reply:
I wanted to give it a name that would have told me right away that this is the analysis that is needed if the caller was outside the insertion phase. What about drreg_outside_insertion_phase_analysis()?

Reply:
Since the other analysis is also outside of the insertion phase, for the lack of a better name, I've reverted it.